extensions/interactions/base.py

```diff
@@ -133,6 +135,12 @@ def customization_arg_specs(self):
             domain.CustomizationArgSpec(**cas)
             for cas in self._customization_arg_specs]
 
+    @property
+    def path_to_customization_args_content_id(self):
+        """The path to the content_id inside customization_args."""
```

Comment:
Explain this "path" thing in more detail, perhaps with an example -- it's non-standard usage of the word.